models/reviews.models.js

```diff
@@ -45,30 +46,66 @@ exports.fetchCommentsByReviewId = (review_id) => {
     });
 };
 
-exports.fetchReviews = (category) => {
+exports.fetchReviews = ({ category, sort_by, order }) => {
   const validCategoryValues = [
     "euro_game",
     "social_deduction",
     "dexterity",
     "childrens_games",
+    undefined,
   ];
 
+  const validSortByValues = [
+    "title",
+    "designer",
+    "owner",
+    "review_img_url",
+    "review_body",
+    "category",
+    "created_at",
+    "votes",
+    undefined,
+  ];
+
+  const validOrderValues = ["asc", "desc", undefined];
+
+  if (
+    !validCategoryValues.includes(category) ||
+    !validSortByValues.includes(sort_by) ||
+    !validOrderValues.includes(order)
+  ) {
+    return Promise.reject({ status: 400, message: "Bad request" });
+  }
+
   const queryValues = [];
 
   let queryStr = `SELECT reviews.*, COUNT(comments.comment_id)::int as comment_count
   FROM reviews
   LEFT JOIN comments ON reviews.review_id=comments.review_id`;
 
-  if (category) {
-    if (!validCategoryValues.includes(category)) {
-      return Promise.reject({ status: 400, message: "Invalid category" });
-    }
+  if (category !== undefined) {
+    category = category.replace(/_/g, " ");
     queryStr += ` WHERE reviews.category=$1`;
     queryValues.push(category);
   }
 
-  queryStr += ` GROUP BY reviews.review_id
-  ORDER BY reviews.created_at DESC;`;
+  queryStr += ` GROUP BY reviews.review_id`;
+
+  if (sort_by !== undefined) {
```

Comment:
this isn't very dry, we can make the query string more dynamic. If we are handling invalid sort_by queries and order queries before we get to this point, we can be safe in the knowledge that if we don't error out, they must be valid values and we can insert them into our query string :)